### DISCUSSION TITLE
Lesz-e operació a BevFeldData bevallási soraihoz tartozó analitikus adatok lekérdezéséhez?

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#40] [Ötletek - gépi interfész]
Kedves NAV- Szakértők,

A QueryDeclarationDataResponse-ban kapja vissza az adózó a bevallási javaslatot a BevfeldData XML-ben. Ezt az adózó csak akkor tudja felelősséggel jóváhagyni, ha meggyőződött az adóbevallás sorainak helyességéről. Azt pedig úgy tudja megtenni, ha ő maga is kiszámolja ezeket az összegeket és összeveti a kiajánlott bevallás soraival. Ha egyezik jóváhagyja, ha nem, akkor keresnie kell a különbözetet. Ehhez a BevfeldData XML bevallási soraihoz rendelt analitikus adatokra lesz szüksége az adózónak. 

Azt szeretném kérdezni, hogy lesz-e olyan operáció, amivel ez lekérdezhető?

Előre is köszönöm!

**Comment by NTCA-HM** [comment]
Kedves @szalaierika !

Mivel az analitika adatot az adózó tölti fel így csak azt tudná vissza kapni amit beküld. Jelenleg nem tervezünk ilyen interfészt.

**Reply by szalaierika** [reply]
Kedves @NTCA-HM !

Köszi szépen a választ. Szerintem az nem probléma, ha indulásra nincs ilyen operáció.

Ha lesz esetleg később, akkor mikorra várható?

Illetve érdemes lenne kétféle módon is lehetővé tenni az egyes bevallási sorokhoz tartozó analitika lekérdezését:

1. Input adat: Minden bevallási sora/ Output adat: minden bevallási sor és a hozzátartozó analitika.
2. Input adat: A bevallás egy sorának azonosítója/ Output adat: a bevallási sor és a hozzátartozó analitika.

:)

**Comment by renced42** [comment]
Kedves @szalaierika 

Megcímkéztem a kérést, későbbi igények közé felvéve.
